Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py

```diff
@@ -1714,8 +1716,12 @@ def create_element_list(arguments_dict: dict) -> tuple[list, list]:
 def get_intelligence_information(client: Client, indicator, ioc_type, intelligence_type):
 
     value = indicator.get('value')
+    remote_api = demisto.params().get('remote_api')
```

Comment:
The value of `remote_api` is either a string of "true" or a string of "false". Both of which would always equal `true` in the following `if remote_api` condition, since they're non-empty strings.

Use CommonServerPython's `argToBoolean` option, and change the parameter type to a checkbox.